tests/math/test_density_matrices.py

```diff
@@ -22,9 +22,13 @@
 
 pytestmark = pytest.mark.all_interfaces
 
+#pylint: disable=redefined-outer-name
```

Comment:
although this is fine, I'd just disable it once at the top of the file. Usually I'd say nbd but I'm sure there are tens or hundreds of test files that start with this and I'd rather set a precedent of less lines here.